MediaBrowser.Common/Json/Converters/JsonStringConverter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace MediaBrowser.Common.Json.Converters
+{
+    /// <summary>
+    /// Converter to allow the serializer to read strings.
+    /// </summary>
+    public class JsonStringConverter : JsonConverter<string>
+    {
+        /// <inheritdoc />
+        public override string Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            return reader.TokenType switch
+            {
+                JsonTokenType.Null => null,
+                JsonTokenType.String => reader.GetString(),
+                _ => GetRawValue(reader)
+            };
+        }
+
+        /// <inheritdoc />
+        public override void Write(Utf8JsonWriter writer, string value, JsonSerializerOptions options)
+        {
+            writer.WriteStringValue(value);
+        }
+
+        private static string GetRawValue(Utf8JsonReader reader)
+        {
+            var utf8Bytes = reader.HasValueSequence
+                ? reader.ValueSequence.FirstSpan
```

Comment:
This won't read the full value if the `ValueSequence` is larger than 1 span, right?

Reply:
Good catch, I must have left that in here when seeing if I could avoid ToArray